nixos/doc/manual/release-notes/rl-1909.xml

```diff
@@ -162,6 +162,18 @@
       which is linked to <literal>fr-toutesvariantes.{aff,dic}</literal>.
     </para>
    </listitem>
+   <listitem>
+    <para>
+     The nginx web server previously started its master process as root
+     privileged, then ran worker processes as a less privileged identity. This
+     was changed to start all of nginx as a less privileged user (defined by
+     <literal>services.nginx.user</literal> and
+     <literal>services.nginx.group</literal>). As a consequence, all files that
+     are needed for nginx to run (included configuration fragments, SSL
+     certificates and keys, etc.) must now be readable by this less privileged
+     identity.
```

Comment:
I would use "user/group" rather than "identity", as that's a word with no specific meaning in this context.

Reply:
Thanks, fixed.